modules/hcp-bm-hc.adoc

```diff
@@ -45,20 +64,20 @@ $ hcp create cluster agent \
     --ssh-key  <path_to_ssh_public_key> \// <7>
     --namespace <hosted_cluster_namespace> \// <8>
     --control-plane-availability-policy HighlyAvailable \// <9>
-    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp_release_image> \// <10>
+    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp_release_image>-multi \// <10>
     --node-pool-replicas <node_pool_replica_count> <11>
 ----
 +
-<1> Specify the name of your hosted cluster, for instance, `example`.
+<1> Specify the name of your hosted cluster, for instance, `my-hosted-cluster`.
 <2> Specify the path to your pull secret, for example, `/user/name/pullsecret`.
-<3> Specify your hosted control plane namespace, for example, `clusters-example`. Ensure that agents are available in this namespace by using the `oc get agent -n <hosted_control_plane_namespace>` command.
+<3> Specify your hosted control plane namespace, for example, `my-hcp-ns`. Ensure that agents are available in this namespace by using the `oc get agent -n <hosted_control_plane_namespace>` command.
```

Comment:
would 'my-hcp-ns' in the example need to be the same as the example above 'my-hosted-ns' instead?